README.md

```diff
@@ -16,8 +16,8 @@ It's designed to work  with PyTorch Lightning
 ## Example
 ```python
 
-from pytorch_lightning_bolts.vaes import VAE
-from pytorch_lightning_bolts.gans import CycleGAN
+from pytorch_lightning import VAE
+from pytorch_lightning import CycleGAN
```

Comment:
wouldn't this be confusing ? what if I don't have bolts installed ?
`from pytorch_lightning.bolts import VAE`

Reply:
this is not updated yet, thx for pointing it our :]

Reply:
shouln't this be `pytorch_lightning.bolts`? And if you don't have bolts installed, that import simply fails